app/assets/src/components/views/compare/SamplesHeatmapVis.jsx

```diff
@@ -161,7 +163,24 @@ class SamplesHeatmapVis extends React.Component {
     this.heatmap.downloadAsPng();
   }
 
+  getMetadataTooltipData(node) {
+    const metadata = node.metadata;
+    return [
+      {
+        name: "Metadata",
+        // TODO (gdingle): display name
+        data: Object.keys(metadata).map(k => [
+          k.replace("_", " ").replace("_v2", ""),
```

Comment:
Wouldn't the second replace fail? if you already replaced '`_`' with '` `'?
What is the use case here? Metadata should have a humanized name, no?